docs/application/native/guides/ui/dali/capture.md

```diff
@@ -0,0 +1,66 @@
+# Capture
+
+
+The Dali::Capture is a class to capture a snapshot of the current scene. The captured result can be saved as a file and rendered by DALi `ImageView` or `ImageVisual`.
+
+
+## How to capture your scene
+
+Applications should follow the example below to use `Dali::Capture`.
+
+``` C++
+Capture capture = Capture::New();
+
+capture.Start( source, size, path, clearColor, quality);
+```
+
+Each parameter of `Capture::Start` is denoted as follows:
+
+ * **Actor source**: source actor to be captured. Only this source actor and its children are captured.
+ * **Vector2 size**: size of captured result image.
+ * **std::string path**: system path to save the captured image file. If the path is empty, `Dali::Capture` does not save the captured image as a file. Instead, the captured image can be rendered by DALi `ImageView` or `ImageVisual`.
+   * `Dali::Capture` supports PNG and JPEG format for the saved image.
+ * **Vector4 clearcolor**: background color of the captured scene
```

Comment:
1) add a period at the end.
2) same question should it be screen?

background color of the captured scene.

Reply:
I thought it is not a sentence so I omitted period. It is necessary?
As I mentioned in above question, it is scene.